lib/util/providers.dart

```diff
@@ -36,7 +36,9 @@ class HumHubNotifier extends ChangeNotifier {
 
   _updateSafeStorage() async {
     final jsonString = json.encode(_humHubInstance.toJson());
+    String lastUrl = _humHubInstance.manifest != null ? _humHubInstance.manifest!.baseUrl : "";
     await _storage.write(key: "hum_hub", value: jsonString);

```

Comment:
In keys names please don't separate the word `humhub`. e.g. `humhub` or `humhub_last_url`. Thanks!

Reply:
Will change the notation to camelCase and move the keys to `const`.